superset-frontend/src/explore/components/DatasourcePanel/DatasourcePanel.test.tsx

```diff
@@ -162,3 +165,45 @@ test('should render a warning', async () => {
     await screen.findByRole('img', { name: 'alert-solid' }),
   ).toBeInTheDocument();
 });
+
+test('should render a create dataset infobox', () => {
+  render(
+    setup({
+      ...props,
+      datasource: {
+        ...datasource,
+        type: DatasourceType.SavedQuery,
```

Comment:
should just be `DatasourceType.Query` not `SavedQuery` they are different models, you'll have to do this for another ticket